add-solution-component/action.yml

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+name: 'copy-environment'
+description: 'Power Platform Copy Environment Action'
+inputs:
+  environment-url:
+    description: 'URL of Power Platform environment to backup; e.g. "https://test-env.crm.dynamics.com"'
+    required: false
+
+  user-name:
+    description: 'Power Platform user name to authenticate with, e.g. myname@my-org.onmicrosoft.com. Setting this input makes user-name and password required; specifying alternate "app-id" credential set of inputs will result in an error.'
+    required: false
+
+  password-secret:
+    description: 'Power Platform password, required if authenticating with username. Do NOT checkin password, instead create a secret and reference it here with: see: https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#using-encrypted-secrets-in-a-workflow'
+    required: false
+
+  app-id:
+    description: 'The application id to authenticate with. Setting this input makes app-id, tenant-id and client-secret required; specifying alternate "username" credential set of inputs will result in an error.'
+    required: false
+
+  client-secret:
+    description: 'The client secret to authenticate with. Required if authenticating with app-id.'
+    required: false
+
+  tenant-id:
+    description: 'Tenant id if using app-id & client secret to authenticate.'
+    required: false
+
+  solution-name:
+    description: 'Name of the solution'
+    required: true
+
+  component:
+    description: 'The schema name or ID of the component to add to the target solution'
+    required: true
+
+  component-type:
+    description: 'The value that represents the solution component that you are adding'
```

Comment:
I know these description fields can't replace docs. But any hint on how/where to get that value?